### ISSUE TITLE
Suggested Try URL results in error

### ISSUE THREAD

**Issue by miketaylr** [vfaronov/turq#9]
I just installed turq-0.2.0.

```
python -m turq
Listening on port 13085
Try http://4.1.168.192.in-addr.arpa:13085/+turq/
```

Visiting in http://4.1.168.192.in-addr.arpa:13085/+turq/ results in an error, but http://localhost:13085/+turq/ works.

**Comment by vfaronov** [comment]
Thank you for this report. Do you perhaps remember what OS this happened on? Was the host configured in any special way to report `4.1.168.192.in-addr.arpa` as its FQDN?

**Comment by vfaronov** [comment]
Anyway, this is hopefully fixed in Turq 0.3.1. By default Turq now detects FQDNs that look like that (IPv4 or IPv6 reverse DNS) and extracts the IP address from them. I don’t want to print just `localhost`, because the point is to guess a URL that can be used for remote access, in case I’m running Turq on some remote machine via SSH.

**Comment by miketaylr** [comment]
> Do you perhaps remember what OS this happened on?

Not 100% sure, it's been a while. Cheers.
